tests/core/engine_adapter/test_integration.py

```diff
@@ -2183,30 +2175,14 @@ def test_dialects(ctx: TestContext):
                 "default": pd.Timestamp("2020-01-01 00:00:00+00:00"),
                 "mysql": pd.Timestamp("2020-01-01 00:00:00"),
                 "spark": pd.Timestamp("2020-01-01 00:00:00"),
+                "databricks": pd.Timestamp("2020-01-01 00:00:00"),
             },
         ),
         (
             "2020-01-01 00:00:00+00:00",
             exp.DataType.build("TIMESTAMP"),
             None,
-            {
-                "default": pd.Timestamp("2020-01-01 00:00:00"),
-                # Databricks' timestamp type is tz-aware:
-                # "Represents values comprising values of fields year, month, day, hour, minute, and second,
-                # with the session local time-zone.
-                # The timestamp value represents an absolute point in time."
-                # https://docs.databricks.com/en/sql/language-manual/data-types/timestamp-type.html
-                #
-                # They are adding a non-aware version TIMESTAMP_NTZ that's currently in public preview -
-                # you have to specify a table option to use it:
-                # "Feature support is enabled automatically when you create a new Delta table with a column of
-                # TIMESTAMP_NTZ type. It is not enabled automatically when you add a column of
-                # TIMESTAMP_NTZ type to an existing table.
-                # To enable support for TIMESTAMP_NTZ columns, support for the feature must be explicitly enabled for
-                # the existing table."
-                # https://docs.databricks.com/en/sql/language-manual/data-types/timestamp-ntz-type.html
-                "databricks": pd.Timestamp("2020-01-01 00:00:00+00:00"),
```

Comment:
@eakmanrq on the Databricks Runtime 15.3 (which is the runtime i'm using for CircleCI to run the tests on), this override didnt seem to be needed anymore.

The default `{"default": pd.Timestamp("2020-01-01 00:00:00")}` seems to work now.

However, have I inadvertently targeted a Databricks runtime that we dont support?

Reply:
I think it is fine then to remove this override